sktime/tests/tests/test_test_utils.py

```diff
@@ -1,10 +1,43 @@
 """Tests for the test utilities."""
 
-from sktime.tests._config import EXCLUDE_ESTIMATORS
+from sktime.registry import all_estimators
+from sktime.tests._config import (
+    EXCLUDE_ESTIMATORS,
+    EXCLUDE_SOFT_DEPS,
+    EXCLUDED_TESTS_BY_TEST,
+)
 from sktime.tests.test_switch import run_test_for_class
 from sktime.utils.dependencies import _check_estimator_deps
 
 
+def test_excluded_tests_by_test():
+    """Test that EXCLUDED_TESTS_BY_TEST contains estimators with <2 test params."""
+    all_ests = all_estimators()
+    filtered_estimators = [
+        x[0]
+        for x in all_ests
+        if (
+            (
+                len(x[1].get_test_params()) < 2
+                or isinstance(x[1].get_test_params(), dict)
+            )
+            and (
+                len(x[1].get_param_names())
+                - len(x[1].get_class_tags().get("reserved_params", []))
+                > 0
+            )
+        )
+    ]
+    excluded_estimators = next(iter(EXCLUDED_TESTS_BY_TEST.values()))
```

Comment:
If you agree to change it, then I suggest to be explicit (and easily understandable).

```suggestion
    excluded_estimators = EXCLUDED_TESTS_BY_TEST["test_get_test_params_coverage"]
```

Reply:
Was trying not to key it to allow for flexibility later on if anything ever changes.